core/Testo.ml

```diff
@@ -142,6 +142,7 @@ let update_id (test : t) =
   let internal_full_name = T.recompute_internal_full_name test in
   let md5_hex = internal_full_name |> Digest.string |> Digest.to_hex in
   assert (String.length md5_hex = 32);
+  (* nosemgrep: ocamllint-str-first-chars *)
```

Comment:
I still think it's better to use Str.first_chars ... it's more readable, but whatever.

Reply:
ok, but it's not reasonable to depend on another regexp library just for this.